src/wallet/states/responding.ts

```diff
@@ -15,6 +15,9 @@ interface AcknowledgeChallenge extends AdjudicatorExists {
   type: typeof ACKNOWLEDGE_CHALLENGE;
   stage: typeof RESPONDING;
 }
+export function acknowledgeChallenge<T extends AdjudicatorExists>(params: T): AcknowledgeChallenge {
+  return { type: ACKNOWLEDGE_CHALLENGE, stage: RESPONDING, ...adjudicatorExists(params) };
```

Comment:
Should we be setting displayMode here so the wallet will pop up for the user to acknowledge the challenge?

Reply:
Yes. Good spot.